src/utils.ts

```diff
@@ -361,3 +362,18 @@ export const nFormatter = (num: number, digits: number): string => {
     (num / si[index].value).toFixed(digits).replace(rx, "$1") + si[index].symbol
   );
 };
+
+export const getVersion = () => {
+  const version = document.querySelector('meta[name="version"]');
```

Comment:
Let's not query the DOM for this. Can we pass through the version number the same way it was added to the DOM?